src/test/resources/org/jabref/logic/xmp/article_dublinCore.bib

```diff
@@ -7,6 +7,7 @@ @Article{Olly2018
   number       = {1},
   pages        = {1-2},
   month        = mar,
+  day          = {1},
```

Comment:
Could you add a new test case please?

One with `day` and one without `day`? - Both case might appear in the wild and should be tested